cli/standalone/mod.rs

```diff
@@ -660,24 +661,28 @@ pub async fn run(
   };
 
   let permissions = {
-    let mut permissions =
-      metadata.permissions.to_options(maybe_cwd.as_deref())?;
+    let mut permissions = metadata.permissions.to_options()?;
```

Comment:
I'm not sure I'm a fan of not explicitly passing CWD - now it becomes buried very deeply

Reply:
The CWD is not used internally here.